cli/src/torii/routing.rs

```diff
@@ -394,68 +390,36 @@ mod subscription {
 #[cfg(feature = "telemetry")]
 async fn handle_version(sumeragi: Arc<Sumeragi>) -> Json {
     use iroha_version::Version;
-    /*
-        TODO REDO
-        #[allow(clippy::expect_used)]
-        reply::json(
-            &wsv.blocks()
-                .last()
-                .expect("At least genesis should always exist")
-                .value()
-                .version()
-                .to_string(),
-    )
-         */
-    warp::reply::json(b"Not implemented")
+
+    #[allow(clippy::expect_used)]
+    let string = sumeragi
+        .wsv_mutex_access()
+        .blocks()
+        .last()
+        .expect("At least genesis should always exist")
```

Comment:
Actually, sometimes it doesn't. 

```suggestion
        .expect("Genesis not applied. Nothing we can do. Solve the issue and rerun.")
```

Reply:
yes